server.js

```diff
@@ -22,8 +24,27 @@ app.use(cors())
 app.use(express.json())
 
 // Start defining your routes here
+// This is your first endpoint
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
```

Comment:
This listEndpoints is a really nice touch! I saw that you had imported it and wondered what it was. I'm gonna steel this one ;)